### PR TITLE
Feature/redux observable

### PR DESCRIPTION
**Pull request by HuyAms**

### CODE REVIEW

File: src/modules/epics.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ *
+ */
+
+import {combineEpics} from 'redux-observable'
+import {moduleEpics as postEpics} from '../modules/Post'
+import {moduleEpics as postsEpics} from '../modules/Posts'
+import {appEpics} from '../modules/App'
+
+export const createRootEpic = () => {
+	return combineEpics(...postEpics, ...postsEpics, ...appEpics)
```
Comment (vinhlee95) [inline src/modules/epics.ts:14]: Why do we need to spread those epics? 🤔

File: src/modules/Posts.ts [hunk 2]
```diff
@@ -45,4 +49,5 @@ export const reducer = posts
 // Actions
 // ------------------------------------
 
-export const getPosts = () => moduleActions.getModel()
+export const getPosts = () => getAsync.request({})
+export const cancelPostsRequest = () => getAsync.cancel()
```
Comment (vinhlee95) [inline src/modules/Posts.ts]: I would think `cancelGetPosts` is a better name

File: src/modules/Post.ts [hunk 3]
```diff
@@ -48,4 +57,5 @@ export const reducer = post
 // Actions
 // ------------------------------------
 
-export const getPost = (id: number) => moduleActions.getModel(id)
+export const getPost = (id: string) => getAsync.request({params: id})
+export const cancelPostRequest = () => getAsync.cancel()
```
Comment (vinhlee95) [inline src/modules/Post.ts]: I would think `cancelGetPost` is a better name

File: src/modules/epics.ts (hunk 1 above)
Comment (HuyAms) [reply src/modules/epics.ts:14]: Because those are arrays of epics

**Pull request closed without merging**